modules/dcache/src/main/java/diskCacheV111/services/PoolStatisticsV0.java

```diff
@@ -338,26 +269,34 @@ public void getInfo(PrintWriter pw) {
         pw.println("   Stat Base : "+_dbBase);
         pw.println("   Html Base : "+_htmlBase);
         pw.println("      Images : "+ _images);
-        pw.println("    Next Run : "+_hourly);
+        pw.println("    Next Run : "+ new Date(Calendar.getInstance().getTime().getTime() +
```

Comment:
I think `Instant.now().plusMillis()' does the same, isn't it? 

https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/Instant.html#plusMillis(long)

Reply:
yes, it does! But the output does not look the same in the println:
If you use Instant.now().plusMillis(...) it looks like:
2021-09-10T14:14:55.922033900Z
If you use new Date(...) it shows:
Fri Sep 10 16:14:56 CEST 2021

I don't know if this is important, but the "old" cellCron-scheduler printed the second one...

Reply:
This is only the matter of formatter that is used:
```
jshell> DateTimeFormatter.ofPattern("E L HH:mm:ss z yyyy").format(ZonedDateTime.now())
$10 ==> "Fri 9 16:29:34 CEST 2021"
```

Reply:
ohhhh.... okay :D
I will change it.

Reply:
javadoc is your friend :)

https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html